src/bindings/RescriptCompilerApi.res

```diff
@@ -88,15 +87,35 @@ module LocMsg = {
     endColumn: int,
   }
 
-  let decode = (json): t => {
-    open Json.Decode
-    {
-      fullMsg: json->field("fullMsg", string, _),
-      shortMsg: json->field("shortMsg", string, _),
-      row: json->field("row", int, _),
-      column: json->field("column", int, _),
-      endRow: json->field("endRow", int, _),
-      endColumn: json->field("endColumn", int, _),
+  let decode = (json: Js.Json.t): t => {
```

Comment:
Why do we even drop all the existing json decode library code? If the library is only blocking due to uncurried flags, I'd rather vendor the dependency instead and drop the package dep.

Or is there a fundamental issue that prevents the compiler from compiling the lib code?

Reply:
I added JSON lib in .res format